packages/main/src/plugin/kubernetes-context-state.ts

```diff
@@ -291,16 +297,19 @@ export class ContextsManager {
     options: CreateInformerOptions<T>,
   ) {
     informer.start().catch((err: unknown) => {
+      const nextTimeout = options.backoff.get();
       if (err !== undefined) {
-        console.warn('informer start error: ', String(err));
+        console.warn(
+          `Trying to watch ${options.resource} on the kubernetes context named "${context.name}" but got a connection refused, retrying the connection in ${nextTimeout / 1000}s. ${String(err)})`,
```

Comment:
Aren't we gonna have the console spammed with those messages when our cluster is not started (E.g. Kind) ?

Reply:
At startup, you'll get spammed with these logs, but with the backoff retry, you'll get a message every minute only after a couple of minutes

Reply:
Maybe we should use debug ? So if user is trying to find an error in the logs, he do not only see hundreds of those messages after a few hours of running. I feel like, if this is expected to fail, we do not need to log it 

What do you think ?

Reply:
ok, let's change to debug